core/vm/interpreter_zkevm.go

```diff
@@ -14,8 +36,8 @@ func NewZKEVMInterpreter(evm VMInterpreter, cfg Config) *EVMInterpreter {
 	case evm.ChainRules().IsBerlin:
 		jt = &zkevmForkID4InstructionSet
 	}
+	jt = copyJumpTable(jt)
```

Comment:
why this change?

Reply:
I need to clone the original jump table in every case otherwise we end up wrapping and wrapping each time this function is called and get a deeper stack with multiple iterations of the counters being handled

Reply:
please leave a comment then :)